assemblies/static/src/main/resources/resources/sparkl-require-js-cfg.js

```diff
@@ -27,7 +27,7 @@
     prefix = requirePaths['sparkl/components']  = CONTEXT_PATH + 'api/repos/sparkl/resources/amd-components';
 
   } else if(typeof FULL_QUALIFIED_URL != "undefined") { // embedded production
-    prefix = requirePaths['sparkl/components']  = FULL_QUALIFIED_URL + 'api/repos/sparkl/resources/amd-componentss';
+    prefix = requirePaths['sparkl/components']  = FULL_QUALIFIED_URL + 'api/repos/sparkl/resources/amd-components';
```

Comment:
This error was likely never detected given in production `CONTEXT_PATH` is defined. It's not clear what's the use case (embedding?). In any case, it's clearly an error.